04/style.css

```diff
@@ -0,0 +1,124 @@
+button {
+    display: block;
+    margin: 20px auto; 
+}
+
+/* hamburger start */
+.hamburger {
+    background: transparent;
+    border: 0;
+    border-top: 5px solid black;
+    height: 35px;
+    position: relative;
+    transition: 0.3s transform linear;
+    width: 40px;
+}
+
+.hamburger::before, .hamburger::after {
+    border-top: 5px solid black;
+    content: "";
+    left: 0;
+    position: absolute;
+    top: 0;
+    transform: translateY(10px);
+    width: 100%;
+}
+
+.hamburger::after {
+    transform: translateY(25px);
+    transition: 0.3s transform linear;
+}
+      
+.hamburger:focus {
+    border: none;
+    transform: rotate(45deg) translateY(5px);
+}
+
+.hamburger:focus::after {
+    transform: rotate(-90deg) translateX(-10px);
+}
+/* hamburger end*/
+
+
+nav {
+    display: none;
+    padding: 10px;
+}
+
+button:focus + nav {
+display: block;
+}    
+
+li {
+    background-color: black;
+    margin: 5px;
+}
+
+a {
+    color: white; 
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-size: 50px;
+    text-decoration: none;  
+}
+
+li > ul {
+    display: none;
+}
+
+li:hover>ul {
+    display: block;
+}
+
+li > ul > li {
+    margin-left: 10%;   
+}
+
+a + ul > li > a {
+    font-size: 40px; 
+ }
+
+
+@media (min-width:600px) {
+    button {
+        display: none;
+    }
+
+    nav {
+        display: block;
+        background-color: black;
+    }
+
+    nav > ul {
+        display: flex;
+    }
+
+    ul > li {
+        margin-left: 20px;
+        margin-right: 20px;
+    }
+   
+    li:nth-child(2){
+        position: relative;
+    }
+
+    li > ul {
+        display: none;
+        position: absolute;
+        top: calc(100% + 15px);
```

Comment:
Niestety to powoduje, że jeśli zjeżdżamy powoli po menu to zagnieżdżenie znika.
Rozwiązaniem będzie zrobienie odstępu dzięki paddingowi dla li :)